.github/workflows/test.yml

```diff
@@ -19,6 +19,11 @@ jobs:
       - name: Checkout sources
         uses: actions/checkout@v4
 
+      - name: Install package with dependencies
+        uses: ./.github/actions/install-pkg
+        with:
+          python-version: ${{ matrix.python-version }}
+
```

Comment:
The subsequent install uv and setup nox aren't needed anymore, correct? (lines 27-36)

Reply:
Uhhh. doh. Thanks :laughing:

Reply:
ad72de8